src/lib/docs/Templates.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Templates Overview
+description: An overview of the available WharfKit templates, designed to accelerate your blockchain development with Antelope.
+category: Templates
+published: true
+slug: index
+---
+
+# WharfKit Templates Overview
+
+WharfKit provides a suite of templates which empower developers to easily create custom plugins that enhance and extend the capabilities of the WharfKit ecosystem.
+
+Here's an overview of the available templates:
+
+## [Account Creation Plugin Template](/docs/templates/account-creation-plugin-template)
+
+This template can be used as a starting point for developers looking to create [account creation plugins](/docs/session-kit/plugin-account-creation) that handle account creation. Through those plugins, developers can make any account creation service usable with the [Wharf SessionKit](/docs/session-kit).
+
+## [Wallet Plugin Template](/docs/templates/wallet-plugin-template)
+
+This template can be used as a starting point for developers looking to create [wallet plugins](/docs/session-kit/plugin-wallet). Through those plugins, developers can make any Antelope wallet compatible with the [Wharf SessionKit](/docs/session-kit).
+
+## [Login Plugin Template](/docs/templates/login-plugin-template)
+
+This template can be used as a starting point for developers looking to create [login plugins](/docs/session-kit/plugin-login). These plugins can be used to extend the capabilities of the [Wharf SessionKit](/docs/session-kit) by adding hooks before or after users login.
+
+## [Transact Plugin Template](/docs/templates/transact-plugin-template)
+
+This template can be used as a starting point for developers looking to create [transact plugins](/docs/session-kit/plugin-transact). These plugins can be used to extend the capabilities of the [Wharf SessionKit](/docs/session-kit) by manipulating transactions before or after they are signed and broadcasted.
```

Comment:
Transaction manipulation can only be done in the `beforeSign` plugin - so this probably needs to be changed.

Maybe: 

> These plugins can be used to extend the capabilities of the [SessionKit](/docs/session-kit) to allow developers to customize the logic of the [Transact](#) method before signing, after signing, and after broadcasting.

Reply:
lol makes sense 😅